ccc/secrets_server.py

```diff
@@ -39,38 +56,61 @@ def from_env(
                 ))
         cache_file = os.environ.get(cache_file, None)
 
+        secret = get_secret_cfg_from_env_if_available()
+
         return SecretsServerClient(
-                endpoint_url=os.environ.get(endpoint_env_var),
-                concourse_secret_name=os.environ.get(concourse_secret_env_var),
-                cache_file=cache_file
+            endpoint_url=os.environ.get(endpoint_env_var),
+            concourse_secret_name=os.environ.get(concourse_secret_env_var),
+            cache_file=cache_file,
+            secret=secret,
         )
 
     @staticmethod
-    def default():
+    def default(encryption:bool=False):
```

Comment:
hmmm..... actually, I would prefer to already prepare the incoming change to multiple secrets

--> please factor-out the calculation of the secrets-path and add this path as an argument (of course, keep previous behaviour, i.e.: if no path is given, fallback to default path